core/domain/wipeout_service.py

```diff
@@ -233,21 +240,21 @@ def _generate_activity_to_pseudonymized_ids_mapping(activity_ids):
     }
 
 
-def _delete_improvements_models(user_id):
+def _delete_models(user_id, module_name):
     """Delete the improvements models for the user with user_id.
```

Comment:
Please fix the docstring. (And, in the future, always make sure to check the docstring when you modify any method/function.)